apps/web/components/SubpageMainContent/SubpageMainContent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react'
+import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
+import { theme } from '@island.is/island-ui/theme'
+import {
+  Box,
+  GridContainer,
+  GridRow,
+  GridColumn,
+} from '@island.is/island-ui/core'
+
+interface SubpageMainProps {
+  main: ReactNode
+  image?: ReactNode
+}
+
+export const SubpageMainContent: FC<SubpageMainProps> = ({ main, image }) => {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = React.useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
```

Comment:
we should try using media queries for break points instead of js (whenever possible)

Reply:
removed